webpack/app/containers/DeckManagementContainer.jsx

```diff
@@ -4,7 +4,7 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 import { requestOwnDecks } from 'actions/deckManagementActions';
-import DeckThumbnail from '../presentationals/deckManagement/DeckThumbnail';
+import { DeckThumbnail } from '../presentationals/deckManagement/DeckThumbnail';
```

Comment:
You can just use 'presentationals/deckManagement/DeckThumbnail'
Also, maybe it's best to place all this under presentationals/components